lib-injection/min_compatible_versions.csv

```diff
@@ -62,18 +63,18 @@ elasticsearch8,~=8.0.1
 elasticsearch[async],0
 envier,==0.5.2
 exceptiongroup,0
-falcon,~=3.0
+falcon,~=3.0.0
 fastapi,~=0.64.0
 flask,~=0.12.0
 flask-caching,~=1.10.0
-flask-login,~=0.6.2
+flask-openapi3,0

```

Comment:
This one seems off

Reply:
cc @emmettbutler

Reply:
https://pypi.org/project/flask-openapi3/#history

Reply:
Hmmm... I think this means we don't specify a min version, we just test with `latest` as per our riotfile

Reply:
oh! yeah, that was difficult to read, I didn't realize the name had `3` in it!